ajax/libs/mapbox-gl/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "mapbox-gl",
+  "description": "A WebGL interactive maps library",
+  "filename": "mapbox-gl.js",
+  "license": "BSD-3-Clause",
+  "homepage": "https://www.mapbox.com/mapbox-gl-js/api/",
```

Comment:
This page get 404 error.
Maybe we should turn to use https://www.mapbox.com/maps/?